README.md

```diff
@@ -36,13 +36,27 @@ xedgewarp is available in the AUR as [xedgewarp-git](https://aur.archlinux.org/p
 
 xedgewarp is make-based. Hence, you can clone the git repository and compile and install it via
 
-````
+```
 git clone https://github.com/Airblader/xedgewarp
 cd xedgewarp
 make
 sudo make install
 ```
 
+### Dependencies
+
+xedgewarp depends upon varios X11 libraries, including:
+ * bits from XCB for X11; [libxcb, libX11-xcb](https://xcb.freedesktop.org/)
+ * Input Extensions for X11; [libXi](https://xorg.freedesktop.org)
+ * Display Manager Control Protocol Library for X11; [libXdmcp](https://xorg.freedesktop.org)
+ * Miscellaneous Extension Library for X11; [libXext](https://xorg.freedesktop.org)
+ * bits from X11; [libx11](https://xorg.freedesktop.org)
+
+For Debian and apt/dpkg managed distributions:
```

Comment:
I don't like maintaining a long list of dependency packages in the README for all distributions. I'd prefer if somebody created a package instead which we can then link to – in general I'd like to avoid encouraging people to compile from source as this only brings up a lot of the same questions.

I'll happily link a package from the README (like I did for Manjaro / Archlinux).